src/Hl7.Fhir.Support.Poco/FhirPath/FhirEvaluationContext.cs

```diff
@@ -33,12 +33,12 @@ public FhirEvaluationContext(ITypedElement resource, ITypedElement rootResource)
         }
 
         public FhirEvaluationContext(ScopedNode resource)
-            : base(toNearestParentResource(resource))
+            : this(toNearestParentResource(resource))
         {
             RootResource = Resource is ScopedNode node ? node.ResourceContext : resource;
         }
 
-        private static ScopedNode toNearestParentResource(ScopedNode node) => node.AtResource ? node : node.ParentResource;
+        private static ITypedElement toNearestParentResource(ScopedNode node) => node.AtResource ? node : node.ResourceContext;
```

Comment:
The name of this function is now pretty misleading ;-)

Reply:
I think you looked at the a previous commit. It is called now `toNearestResource`. See https://github.com/FirelyTeam/firely-net-common/blob/cf4db7cbd5475647abd1bde4725e8fef981ebdf1/src/Hl7.Fhir.Support.Poco/FhirPath/FhirEvaluationContext.cs#L45